app/views/recipes/_recipe.html.erb

```diff
@@ -1,32 +1,31 @@
-<div id="<%= dom_id recipe %>">
-  <p>
-    <strong>Name:</strong>
+<div class="recipe-details" id="<%= dom_id recipe %>">
+  <p class="recipe-detail">
+    <strong class="label">Name:</strong>
     <%= recipe.name %>
   </p>
 
-  <p>
-    <strong>Preparationtime:</strong>
+  <p class="recipe-detail">
+    <strong class="label">Preparation Time:</strong>
     <%= recipe.preparationTime %>
   </p>
 
-  <p>
-    <strong>Cookingtime:</strong>
+  <p class="recipe-detail">
+    <strong class="label">Cooking Time:</strong>
     <%= recipe.cookingTime %>
   </p>
 
-  <p>
-    <strong>Description:</strong>
+  <p class="recipe-detail">
+    <strong class="label">Description:</strong>
     <%= recipe.description %>
   </p>
 
-  <p>
-    <strong>Public:</strong>
+  <p class="recipe-detail">
+    <strong class="label">Public:</strong>
     <%= recipe.public %>
   </p>
 
-  <p>
-    <strong>User:</strong>
+  <p class="recipe-detail">
+    <strong class="label">User:</strong>
     <%= recipe.user_id %>
   </p>
-
```

Comment:
- [ ] I notice that when a user is creating a new recipe, the last part of the form has this `user` and initially I wasn't sure what to put there so I put my name which threw this error below and I later realized that a user is supposed to input their ID in there. But I don't think this is a good way to create a recipe because what happens if a user doesn't know that they are `ID number 1 or 2 ` then that means they can't create a recipe. So I will suggest that you get the `current_user's ID` and use that in the creation of a recipe since whoever is creating the recipe is going to be logged in before even getting access to that page as a result the `ID` will be `added by default` for the user :wink: 
- Error Message
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/76c6893a-8443-4d37-83a4-40e94144f340)

- Your Form Now
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/11b17a04-8b3a-41c2-b8d8-0029811251c0)

Reply:
Fixed.